source/f_problem_solving.rst

```diff
@@ -5,11 +5,7 @@ FAQ and Problem Solving
 
 This section will help you solve some problems or errors that may occur when using 3Di.
 Some issues are due to the software, these will be summarised in the section Known Issues, including a temporary solution.
-Errors, due to input data or other user settings can occur in various components or steps of the modeling process:
-
-#. Generating a 3Di model from the spatialite data or
-
-#. During simulation via the live site or via an API call.
+Errors, due to input data or other user settings are usually covered by the schematisation and raster checker. In rare cases however they might not catch a potential issue. On this page we cover the known issues with 3Di.
```

Comment:
Ik vind deze uitleg onduidelijk. Ik denk dat het duidelijker wordt als er staat: Op deze pagina vind je _x_. Voor _y_, zie _hier_.

Ook het onderscheid wat er op deze pagina gemaakt wordt tussen "Known issues" en "Frequently endured issues" vind ik onduidelijk. Voorstel: alles bij elkaar zetten onder 'Known issues', waarbij we zo mogelijk de bekendste issues bovenaan zetten.